platform/darwin/src/MGLGeometry.h

```diff
@@ -6,6 +6,30 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+
+typedef double MGLLocationRadians;
+typedef double MGLRadianDistance;
+typedef double MGLRadianDirection;
+
+extern double const MGLMetersPerRadian;
+
+/** Defines the coordinate by a `MGLRadianCoordinate2D`. */
+typedef struct MGLRadianCoordinate2D {
```

Comment:
Let’s move all this new code to MGLGeometry_Private.h. MGLRadianCoordinate2D is only useful for an internal calculation, but it isn’t used in any public API. I think its presence would confuse developers who are looking for normal coordinates to use with various public APIs.